format_and_overlap_determination.md

```diff
@@ -0,0 +1,118 @@
+# Description of Format and Overlap Determinations #
+This attempts to describe what this system actually does. This does not attempt to exhaustively describe all corner cases or emergent properties.
+
+## Definitions ##
+**Format**
+: mono (spm), multi (mpm), serial (ser)
+
+**Cluster**
+: Collection of HT Items, Holdings, and Concordance pairs united by a list of OCNs determined to be equivalent.
+
+**Bib Record**
+: A preferred record from the Zephir clustering system containing one or more HT Items. Has a single Hathitrust record ID.
+
+**HT Item**
+: Volume ID and associated metadata taken from the Hathifiles. 
+
+**Print Serials List**
+: List of Record Ids determined to be serials by some unknown UMich process. List is limited to those held by UMich.
+
+**Large Clusters**
+: Clusters large enough that they create implementation problems. Identified by an OCN, there are currently 11 on the list.
+
+**Billing Entity**
+: The `billing_entity` field taken from the `ht_collections` table associated with a particular HT Item through the collection code. Used for cost allocation when holdings are incomplete or no clustering can be performed. Also has subtle and likely inconsequential implications for items contributed by UCM and KEIO.
+
+**n_enum**
+: A normalized enumeration. Description fields are processed for enumeration and chronology. Only enumeration ends up being used. 
+
+
+## Item Format ##
+
+**Serial**
+: Items with a **Bib Record ID** found on the **Print Serials List**, or in a **Large Cluster** 
+
+**Multi-part Monograph**
+: Is not a Serial. Shares a **Bib Record** with an item that has a non-empty `n_enum`. [^1]
+
+**Single-part Monograph**
+: Is not a Serial or Multi-part
+
+**NB**: A cluster may contain items of multiple formats.  
+**NB2**: Format as reported by members is irrelevant.
+
+## Cluster Format ##
+
+**Multi-part Monograph**
+: Any of the items in the cluster are MPMs.
+
+**SER / SPM**
+: Is not MPM. One or more of the items in the cluster are SPM _and_ one or more of the items in the cluster are SER.
+
+**SER**
+: Is not MPM or SER/SPM. Any of the items in the cluster are SER (and by implication ALL are SER).
+
+**SPM**
+: Is not MPM, SER/SPM, or SER.
+
+## Cost Allocation ##
+
+**Target Cost**
+: Set by the Board and Executive Director. Portion of the budget to be paid for by item allocations.
+
+**Cost Per Volume**
+: Target Cost / Total Number of HT Items
+
+**Member Weight**
```

Comment:
Is this the same as the "tier"?  If not we should add what a tier is.

Reply:
It's the same thing.

Reply:
Well. The **Member Tier** is a label that exists outside the system. 

Melissa will tell me "Please make an estimate for member xyz, they are a US institution and tier 1". She determines tier based on the IPEDS. From her boilerplate that goes out to prospective members: "Each member is assigned to one of three tiers based on the member's Total Library Expenditures as reported in public sources."

Each tier (1,2,3) has a weight associated. This association is not codified, other than perhaps in Confluence, but as far as I'm concerned lives in our minds.
Tier 1: 0.67
Tier 2: 1.00
Tier 3: 1.33

Then there are other members that fall outside the tier system, with a weight of 0.00 or above 1.33. These are:

`+------------+--------+`
`| member_id  | weight |`
`+------------+--------+`
`| ucmerced   |   0.00 |`
`| ucsf       |   0.00 |`
`| hathitrust |   0.00 |`
`| utexas     |   3.00 |`
`| flbog      |   5.00 |`
`| usg        |   3.00 |`
`+------------+--------+`

Reply:
Those non-tier weights were, I believe, rather arbitrarily cobbled together, probably by Mike.

Reply:
ooookay, this is where the system thing from my comment above gets worked out.

Reply:
My impression was that the weights (including for the system members, which I think is based on the number of R1 research schools in the system) are encoded in the billing model which has been approved by the Board, so I would not classify them as "arbitrary". It would be worth checking with Mike that this matches his understanding, but yes, I think that given our current billing model this is the right way to do it.

Reply:
The system weights come from this: https://www.hathitrust.org/sites/www.hathitrust.org/files/member-criteria-formalize.pdf which Natalie quoted from below

Reply:
The tier to weight mapping for Tier 1, 2, 3 is in a policy here:
https://www.hathitrust.org/sites/www.hathitrust.org/files/fee-model-change-proposal.pdf

@nfulkers  will add info about systems & specific R1 schools in Confluence

Reply:
We should add these two links above ^^ to this documentation